frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.scss

```
@@ -0,0 +1,7 @@
+.odc-monitoring-dashboard-dropdown-options {
```

Follow BEM notation:
```suggestion
.odc-monitoring-dashboard__dropdown-options {
```